example_files/mappers/Baseline.rb

```diff
@@ -372,18 +372,8 @@ def create_osw(scenario, features, feature_names)
             end
             
             floor_height = 10
-            # Map system type to openstudio system types
```

Comment:
this doesn't look right...this bit of code 375-386 should not get removed.

Reply:
Fascinating. Must have been a branch thing, as I had copied the file from the example project. In this commit I copied `example_project.json`, `base_workflow.osw`, and `Baseline.rb` from the example project to here. Only Baseline.rb changed, restoring those lines that shouldn't have been deleted.